storage/monitoring.go

```diff
@@ -0,0 +1,17 @@
+package storage
+
+import (
+	logger "github.com/ElrondNetwork/elrond-go-logger"
+	"github.com/ElrondNetwork/elrond-go/core"
+	"github.com/ElrondNetwork/elrond-go/core/atomic"
+)
+
+var log = logger.GetOrCreate("storage")
+
+var cummulatedSizeInBytes atomic.Counter
+
+// Question for review: keep this or remove it (helps us to compute planned memory at runtime)?
```

Comment:
I didn't quite get how this should work if we keep it, but I only checked the code here in this PR.

Reply:
Was moved to `storageUnit.NewCache()` while working on this fix-after-review.